src/io/flutter/view/WidgetPerfTable.java

```diff
@@ -149,6 +148,7 @@ public void onFrame() {
 
   private void updateIconUIAnimations() {
     int lastActive = -1;
+    // TODO(devoncarew): We've seen NPEs from here.
```

Comment:
@jacob314 I wasn't able to track this NPE down. If I remember correctly, it happened early in the connection lifecycle.

Reply:
I have a fix. I'll send out the fix soon.

Reply:
Overall this looks really good. @InMatrix for thoughts on breaking out the Rebuild stats in their own column.
Can you add a screenshot of what the ui looks like if the user pins the perf tab to the right side of the screen?